<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioRecorder</class>
 <widget class="QMainWindow" name="AudioRecorder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AudioRecorder</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QToolButton" name="recordButton">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>250</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>56</width>
      <height>19</height>
     </size>
    </property>
    <property name="text">
     <string>Record</string>
    </property>
   </widget>
   <widget class="QToolButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Layout"/>
   </widget>
   <widget class="QToolButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>218</x>
      <y>250</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>291</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="DeviceComboBox"/>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="CodecComboBox"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Input Device:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Sample Rate:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Sample Size:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QComboBox" name="SampleRateComboBox"/>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QComboBox" name="SampleSizeComboBox"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Sample Type:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QComboBox" name="SampleTypeComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Audio Codec:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Endianness:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="QComboBox" name="EndiannessComboBox"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Channels:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QComboBox" name="ChannelsComboBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>recordButton</zorder>
   <zorder>pauseButton</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>saveButton</zorder>
   <zorder>layoutWidget</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
